libs/orchestrator-ui-components/src/lib/components/Table/DataGridTable.stories.tsx

```diff
@@ -63,9 +63,9 @@ const TableWithEvents = (args) => {
     );
 };
 
-const Story: Meta<typeof Table> = {
+const Story: Meta<typeof DataGridTable> = {
```

Comment:
Currently there is a story in progress to implement Storybook, therefore I did not add any storybooks yet, as this is not the final implementation of storybook.

Reply:
@mts : Some background info here: we discussed Storybook a while ago and decided that we, for now, didn't want to riddle the lib with a storybook. Instead we want to use the mono repo tooling for that

Rationale:
- The storybook could exists for a Surf and a GPL variant and maybe even for the example apps.
- It feels messy to have the storybook deps in the lib: whilst the mono repo has nice tools for it
- Some of these storybooks would need to be published in different places

Reply:
- Very well. I assume it's depending on @tjeerddie's [PR #82](https://github.com/workfloworchestrator/orchestrator-ui/pull/82)
- I can accept to complement Jest snapshot tests and Storybook stories together when this PR is merged since stories in Component Story Format (CSF) will be reused to save snapshots as well.